packages/@aws-cdk/pipelines/test/codepipeline/codepipeline.test.ts

```diff
@@ -192,6 +192,36 @@ test('CodeBuild action role has the right AssumeRolePolicyDocument', () => {
   });
 });
 
+test('CodePipeline enables key rotation on cross account keys', ()=>{
+  const pipelineStack = new cdk.Stack(app, 'PipelineStack', { env: PIPELINE_ENV });
+  const repo = new ccommit.Repository(pipelineStack, 'Repo', {
+    repositoryName: 'MyRepo',
+  });
+  const cdkInput = cdkp.CodePipelineSource.codeCommit(
+    repo,
+    'main',
+  );
+
+  new CodePipeline(pipelineStack, 'Pipeline', {
+    enableKeyRotation: true,
+    crossAccountKeys: true, // requirement of key rotation
```

Comment:
Given this dependency, this should be enforced in code. If someone adds rotation without setting this to true, what's the behavior? We don't want a deploy time failure or a silent failure here.

Reply:
This is already enforced by the underlying `Pipeline` construct. Will add a unit test to capture that.